flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -52,6 +52,24 @@ public SqlParserFixture fixture() {
     @Test
     void testShowCatalogs() {
         sql("show catalogs").ok("SHOW CATALOGS");
+
+        // ----------- The following cases are used to detect LIKE -----------
+
+        sql("show catalogs like '%'").ok("SHOW CATALOGS LIKE '%'");
+        sql("show catalogs not like '%'").ok("SHOW CATALOGS NOT LIKE '%'");
+
+        // ----------- The following cases are used to detect ILIKE -----------
+
```

Comment:
it's better to remove this empty line

otherwise one might think (including me) that this comment relates to all the rest lines in this test